File: help/pt_pt/exercicios_da_ficha_4/Ficha04_004/Ficha04_004.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha04_004 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Número \t: " );
        int num = Integer.valueOf(keyb.nextLine());
        if(num > 0 ){
            System.out.print("Positivo" );
        }else{
            if(num == 0 ){
                System.out.print("Nulo" );
            }else{
                System.out.print("Negativo" );
            }
        }
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 15:31

File: help/pt_pt/exercicios_da_ficha_4/Ficha04_004/Ficha04_004.cpp
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <iostream>
#include <cstring>
#include <cmath>
using namespace std;

// Class definition
class Ficha04_004 {

    public:
        int main() {
            cout << "Número \t: ";
            int num;
            cin >> num;
            if(num > 0 ){
                cout << "Positivo";
            }else{
                if(num == 0 ){
                    cout << "Nulo";
                }else{
                    cout << "Negativo";
                }
            }
            return 0;
        }
};// end of class Ficha04_004
// Execute main function
int main(){
    Ficha04_004 application; // create object
    application.main(); // execute object
    return 0;
}

//C++  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 15:31
